pom.xml

```diff
@@ -67,12 +67,23 @@ THE SOFTWARE.
             <url>https://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+    
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.jenkins.tools.bom</groupId>
+                <artifactId>bom-weekly</artifactId>
```

Comment:
hmm should probably use the latest lts line here, we don't generally track weekly i.e. updating on every version, this may cause issues but I guess not too likely since it's only ionicons api we depend on here...